components/docs-chef-io/content/automate/servicenow_event_creation.md

```diff
@@ -0,0 +1,120 @@
++++
+title = "ServiceNow Event Creation App"
+
+draft = false
+
+gh_repo = "automate"
+
+[menu]
+  [menu.automate]
+    title = "Event Creation App"
+    identifier = "automate/integrations/servicenow/servicenow_event_creation.md Chef Automate Event Creation"
+    parent = "automate/integrations/servicenow"
+    weight = 20
++++
+
+The Chef Automate Event Creation App is a ServiceNow-certified app available in the [ServiceNow](https://store.servicenow.com) store. The Event Creation App integrates Chef Automate with ServiceNow by enabling you to create events and alerts corresponding to the failed Chef Infra client runs and compliance scans. These events and alerts generated in the ServiceNow platforms are due to the failures in the Chef Automate infrastructure automation updates and automated compliance checks. Thus, this app captures the failures in your automated infrastructure and improves your event tracking and resolution.
+
+The Event Creation App exposes the REST API endpoint for communication between Chef Automate and the ServiceNow instance. Chef Automate sends HTTPS JSON notifications to the Event Creation App in a ServiceNow instance to create and update events.
+
+## User Requirements
+
+- Your unique ServiceNow URL. It has the format: `https://ven12345.service-now.com`.
+- Setting up the Event Creation App requires the `x_chef_automate.api` role in configuring it. Your ServiceNow administrator can enable this for you.
+
+## System Requirements
+
+- The [Integration App]({{< relref "servicenow_integration" >}}) is already installed and configured.
```

Comment:
Is the integration app really required for event creation app to work ?

Reply:
@vivekshankar1: will talk to you tomorrow and we can take a call accordingly.